### ISSUE TITLE
seven segment lab

### ISSUE THREAD

**Issue by jamilkhalaf** [byuccl/digital_design_colab#112]
[reference](https://colab.research.google.com/github/byuccl/digital_design_colab/blob/master/Labs/seven_segment_lab/seven_segment_lab.ipynb#scrollTo=yBUsK8h1iaS3)

The initial segment of the lab, focusing on assigning data values to segments and converting binary to hex, is presented in a clear and straightforward manner. However, what seems to be missing in this lab is an elucidation of its practical application or utility. The final section discussing the top module is somewhat unclear, and the purpose behind this particular aspect remains ambiguous. I am unable to see the purpose of this component based on the information provided, so an explanation explaining the real use of such code would be helpful.

**Comment by NyloxtheBard** [comment]
I agree with Jamil. This lab seems relatively basic at first glance, but going through it again, there is a lot that is unclear. One thing that could be clarified is how the figure under top module works, and why it is important. Another would be a demonstration as to how the lab works, either through an image or whatnot before running simulation workspace for the user. This lab is more confusing after I have gone through all of the other labs and lessons than before. Even referencing other lessons would be fine, just need more information as to why it is important/useful to do this lab.
